fuel-vm/src/tests/predicate.rs

```diff
@@ -105,10 +124,188 @@ fn get_verifying_predicate() {
             op::ret(0x10),
         ];
 
-        assert!(execute_predicate(predicate, vec![], idx as usize));
+        assert!(execute_predicate(predicate, vec![], idx as usize, false));
     }
 }
 
+#[test]
+fn gas_used_by_predicates_causes_out_of_gas_during_script() {
```

Comment:
Could you move this test to the old place, please to not shuffle the commit history?=)

Reply:
returned, not sure what happened here. I may have copy pasted then removed the wrong one